static/less/search.less

```diff
@@ -56,17 +56,17 @@
         padding: 0 16px 16px;
         flex: 0 1 auto;
         .dropdown-search-keyword {
-            background: #f8f8f8;
-            border-color: #f3f3f3;
```

Comment:
@felixhahnweilheim as I mentioned below, Lucas and I have agreed, that, instead of using functions like darken and whiten, we should swap the color to the closest looking variable. In this case, it would change from `#f3f3f3` to the variable `@backrgound-color-secondary`. As in this case, the backrgound was already adapted to this variable (`@backrgound-color-secondary`), we only need to delete the border-color in this case. 

Please find out, whether we use this color in other places, too, and if it could potentially be changed to a variable instead. 

Thank you! :)